packages/gatsby/lib/query-runner/page-query-runner.js

```diff
@@ -0,0 +1,120 @@
+/***
+ * Jobs of this module
+ * - Ensure on bootstrap that all invalid page queries are run and report
+ *   when this is done
+ * - Watch for when a page's query is invalidated and re-run it.
+ ***/
+
+const _ = require("lodash")
```

Comment:
Just a general style reflection (es6 module vs require):

Sometime we use `import ... from ...` sometime `const ... = require()`, I think es6 module style is better for performance/readibility but more important I think we need to make a choice! (I always ask to myself "what is better for this file?")

So, what we choose?

ref: https://medium.com/the-node-js-collection/an-update-on-es6-modules-in-node-js-42c958b890c